src/main/java/net/neoforged/neoforge/event/village/VillagerTradesEvent.java

```diff
@@ -28,10 +29,20 @@
 public class VillagerTradesEvent extends Event {
     protected Int2ObjectMap<List<ItemListing>> trades;
     protected VillagerProfession type;
+    private final RegistryAccess registryAccess;
 
+    /**
+     * @deprecated Use {@link #VillagerTradesEvent(Int2ObjectMap, VillagerProfession, RegistryAccess)} instead
+     */
+    @Deprecated(forRemoval = true, since = "1.21")
```

Comment:
The deprecation should be since 1.21.1.
```suggestion
    @Deprecated(forRemoval = true, since = "1.21.1")
```